uw-frame-components/portal/widgets/partials/search-with-links.html

```diff
@@ -0,0 +1,27 @@
+<div class="widget-body">
+  <form action='{{ secureUrl }}' target='{{ config.actionTarget }}'>
+    <md-input-container class="md-block" md-no-float>
+      <input type="text" name="{{ config.actionParameter }}" placeholder="{{ config.searchPlaceholder }}">
+      <md-button type="submit" class="md-primary md-icon-button">
+        <i class="fa fa-search"></i>
+      </md-button>
+    </md-input-container>
+  </form>
+  <!-- Links -->
+  <div layout="row">
+    <div class="search-with-links__{{ config.links.length }}">
+      <circle-button ng-repeat="item in config.links"
+                     data-href="{{ item.href }}"
+                     data-target="{{ item.target }}"
+                     data-fa-icon="{{ item.icon }}"
+                     data-disabled="false"
+                     data-title="{{ item.title }}"
+                     data-trunclen="24">
+      </circle-button>
+    </div>
+  </div>
+</div>
+<a class="launch-app-button" ng-href="widget.url" target="{{ config.target }}">
```

Comment:
Could this be more secure if it included [`rel="noopener"`](https://www.jitbit.com/alexblog/256-targetblank---the-most-underestimated-vulnerability-ever/)?